### DISCUSSION TITLE
Problema con DEBUGGING

### DISCUSSION THREAD

**Discussion by codingemmag** [INGCOM-UNRN-P1/cursada-2023#85] [Preguntas y respuestas] [answered]
Empecé a tener un problema con el path de mi workspace, aparentemente es por las los caracteres de las carpetas pero no estoy seguro. Intente eliminar espacios y simbolos como "*" y "-" pero no logro resolver nada y empiezo a pensar que no es ese el problema.

![bsvscode](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/134022702/0d24ae3d-b355-4917-a620-e5aa89ab3df4)

**Comment by martinvilu** [comment, marked as answer]
El símbolo de segundo puede ser el problema, que no es un caracter habitual
